python/pyspark/sql/streaming.py

```diff
@@ -491,6 +490,7 @@ def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
                                     disables `partition discovery`_.
 
         .. _partition discovery: /sql-data-sources-parquet.html#partition-discovery
```

Comment:
So this is wrong right? @HyukjinKwon can you check the python doc and see if the link is corrected?

Reply:
This is fine - it's a way of reusing a link

Reply:
Sorry for my late response.

Reply:
the links are malformed, please check  http://spark.apache.org/docs/3.0.0-preview2/api/python/pyspark.sql.html#module-pyspark.sql.streaming with `partition discovery` content

Reply:
Oh, right. The root directory is different when it's released. Sure, seems like it's best to stick to the absolute links then. The syntax itself is fine as a relative link

Reply:
@yaooqinn can you make a PR to fix these?

Reply:
OK, I'll check it